spec/javascripts/actions/story_spec.js

```diff
@@ -219,4 +223,94 @@ describe('Story Actions', () => {
       expect(fakeDispatch).toHaveBeenCalledWith(Story.updateHighlight(storyId, true));
     });
   });
+
+  describe('confirmBeforeSaveIfNeeded', () => {
+    const story = storyFactory();
```

Comment:
Can we move this to inside a beforeEach?

Reply:
Sure!